examples/treegrid/treegrid-1.html

```diff
@@ -83,18 +77,8 @@ <h1>Treegrid Email Inbox Example</h1>
     The following example demonstrates how the
     <a href="../../#treegrid">treegrid design pattern (TBD)</a>
     can be used to make an interactive tree that enables users to navigate the hierarchical structure of email conversations
-    and also provides the ability to navigate to elements that describe each email, such as subject and sender.
-  </p>
-  <p>
-    This same example can be used in three ways.
-  </p>
-  <ul>
```

Comment:
Not sure we want to eliminate these operating modes; topic of discussion. We could make them three different example pages, but I am not sure that is the best approach in this case.